src/symbolic_dimensions.jl

```diff
@@ -19,7 +18,7 @@ const ALL_VALUES = vcat(
         if k ∉ SYMBOL_CONFLICTS
     )...
 )
-const ALL_MAPPING = NamedTuple([s => i for (i, s) in enumerate(ALL_SYMBOLS)])
+const ALL_MAPPING = NamedTuple([s => UInt8(i) for (i, s) in enumerate(ALL_SYMBOLS)])
```

Comment:
Not sure if this storage optimization (here and in the `SymbolicDimensions` struct) is worth it.

Reply:
I think it's fine to use this. It also helps readability because it indicates there we only expect 256 max indices here (which is true).

I guess you could do UInt16 just in case the user wants to add a bunch of symbols of their own, so they don't run into any weird overflow bugs. But it's not a big deal if you'd prefer UInt8.